server/routes/serviceProviderReferrals/dashboardPresenter.test.ts

```diff
@@ -41,15 +41,19 @@ describe(DashboardPresenter, () => {
           { text: '26 Jan 2021', sortValue: '2021-01-26', href: null },
           { text: 'ABCABCA1', sortValue: null, href: null },
           { text: 'George Michael', sortValue: 'michael, george', href: null },
-          { text: 'Accommodation Services - West Midlands', sortValue: null, href: null },
+          {
+            text: 'Accommodation Services - West Midlands',
+            sortValue: 'accommodation services - west midlands',
```

Comment:
> I believe it's sorted by the text value if this is null

Yep, that's right.

> I wanted to at least keep it consistent with the PP dashboard

Good idea, but with the above in mind it would perhaps be better to make the `sortValue` null on the PP dashboard instead?

Reply:
Yeah, you're right - I did this the wrong way round... have just removed this commit and pushed a replacement.